src/k8s/pkg/utils/experimental/snapdconfig/meta.go

```diff
@@ -33,9 +33,9 @@ func ParseMeta(ctx context.Context, s snap.Snap) (Meta, bool, error) {
 		return Meta{}, false, fmt.Errorf("failed to parse snap config mode: %w", err)
 	}
 
-	// default meta.orb is none
+	// default meta.orb is k8sd
 	if parse.Meta.Orb == "" {
-		parse.Meta.Orb = "none"
+		parse.Meta.Orb = "k8sd"
```

Comment:
that's all you really need, the controller will use this to configure snapd configs after the node is bootstrapped (or joins a cluster).